tools/bigquery-query-plan-exporter/bqutil/bqexplain.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timedelta
+from google.cloud import bigquery
+import argparse
+import json
+import sys
+
+
+class BQQueryPlanExporter(object):
+    def __init__(self, project=None, location='US'):
+        self.project = project
+        self.location = location
+        self.bq = None
+
+    def get_client(self):
+        if self.bq is None:
+            self.bq = bigquery.Client(project=self.project,
+                                      location=self.location)
+        return self.bq
+
+    def get_jobs(self, start_days_ago, end_days_ago):
+        c = self.get_client()
+        dt0 = datetime.utcnow() - timedelta(start_days_ago)
+        dt1 = datetime.utcnow() - timedelta(end_days_ago)
+        return [job._properties for job in c.list_jobs(min_creation_time=dt0,
+                                                       max_creation_time=dt1)
+                if hasattr(job, 'query_plan')]
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Export BigQuery Query Plans.')
+    parser.add_argument('--project', type=str, default=None, nargs='?', help='project id')
```

Comment:
Break these lines up to respect 80 char line length. Run the `pylint` ont this file and correct until passes.